app/javascript/packs/custom_policy.js

```diff
@@ -0,0 +1,11 @@
+import {render} from 'react-dom'
+import React from 'react'
+import {CustomPolicy} from 'Policies/components/CustomPolicy'
+import {safeFromJson} from 'Policies/util'
+
+document.addEventListener('DOMContentLoaded', () => {
+  const container = document.getElementById('custom-policy-container')
+  const jsonPolicy = container.dataset.policy
+  const props = (jsonPolicy) ? {policy: safeFromJson(jsonPolicy)} : {}
+  render(<CustomPolicy {...props} />, container)
```

Comment:
Isn't `policy` the only prop? Maybe we could avoid using spread syntax?

Reply:
Yup, it is, I was trying to avoid sending `policy: {}` and sending just `{}` when there's none sent from Rails, lazy prop loading. Any better suggestion?

Reply:
What I meant is why not simply `<CustomPolicy policy={policy} />`

But I made a suggestion [here](https://github.com/3scale/porta/pull/687/files#r263698878).

Reply:
yeah, changing `safeFromJson` will be the way